files-community/Files
# Feature: Disable warning when a new name is empty

**Resolved / Related Issues**
   Closes #12140

**Removed from**
- New Item Dialog
- New Archive Dialog
- New Shortcut Dialog

**Validation**
How did you test these changes?
   1. Open app
   2. Open the menu to create a new file
   3. See that the warning is not displayed when the name is empty
